### DISCUSSION TITLE
CLI client

### DISCUSSION THREAD

**Discussion by rafidini** [mgpai22/ergpy#10] [Ideas]
It would be interesting for basic operations to be done with a CLI client.

**Examples**

```
$ ergpy get-wallet-address --amount=0 --wallet-mnemonic="xxxxxx" --mnemonic-password="xxxxx"
Your wallet address: XXXXXXXXXX

$ ergpy get-box-info --index=0 --sender-address="xxxxx"
Box Info: XXXXXXXXXX
```
> Examples above are in the terminal using `bash`.

Do you want this feature maybe?

**Comment by mgpai22** [comment]
This is a wonderful feature! I was thinking about it for the future.

**Comment by mgpai22** [comment]
@rafidini 
This could be useful
https://github.com/ladopixel/tERGminal/
